### PR TITLE
Add check for when image build option `repo` is null, drop image resolves #9

### LINKED ISSUE #9: Images overwritten with `null` should be dropped from the deployment environment
The manifest can define all or some images to build but for specific environments you may want to deploy only one or a segment of containers.
```
images: {
	'service1': {
		...
	},
	'service2': null,
	'sqs-service': null,
	'service4': null
},
```

### PR DESCRIPTION
**Pull request by Sieabah**

### CODE REVIEW

File: lib/deployment-tasks.js [hunk 1]
```diff
@@ -88,6 +88,12 @@ function buildImages(registry) {
 	return function(buildOptions) {
 		logger.info('Building Docker Images');
 		logger.info('Will build the following images:');
+
+		for(let name in buildOptions.images)
```
Comment (Sieabah) [inline lib/deployment-tasks.js:92]: _.filter or any type of filter destroys object keys, why I ended up using for..in

**Comment by Sieabah** [comment]
Resolves #9

**Comment by Robert-W** [comment]
Looks good to me

**Review by Robert-W — APPROVED** [review]

**Review by andrewmarcus — COMMENTED** [review]
Could we actually set the entire section to null rather than repo: null?  Or would that not merge properly when composing the environment manifests?

**Comment by Sieabah** [comment]
@andrewmarcus With how _.merge is run through it doesn't look like null falls through in the end. DockerBuildContext is always set and another parameter. I didn't want to rewrite the whole flow of building the options

File: lib/deployment-tasks.js (hunk 1 above)
Comment (bodnarbm) [reply lib/deployment-tasks.js:92]: `_.pickBy` is the equivalent of filter for objects. https://lodash.com/docs/4.17.4#pickBy

Comment (Sieabah) [reply lib/deployment-tasks.js:92]: #nativeLyfe

**Pull request merged**